crates/typst/src/math/ctx.rs

```diff
@@ -202,7 +204,16 @@ impl<'a, 'b, 'v> MathContext<'a, 'b, 'v> {
             let class = self.style.class.as_custom().or(glyph.class);
             if class == Some(MathClass::Large) {
                 let mut variant = if self.style.size == MathSize::Display {
-                    let height = scaled!(self, display_operator_min_height);
+                    let display_operator_min_height = {
+                        // https://github.com/w3c/mathml-core/issues/126#issuecomment-1425804878
+                        let mut height = self.constants.display_operator_min_height();
+                        let min_height = (SQRT_2 * self.ttf.height() as f64) as u16;
```

Comment:
disclaimer: I dont really understant what `self.ttf.height()` is. I also dont understant what is `baseGlyph` in the webkit algorithm.

Reply:
`self.ttf.height()` isn't the correct metric here. it is a font-global height. what the webkit algorithm is doing is taking the height of the concrete glyph (in the bug case the integral). that's the `baseGlyph` ("base" because it's not yet stretched) and it corresponds to the `glyph` variable in our code. so in this case `self.ttf.height()` needs to be replaced by `glyph.height()`. since the glyph's height is already stretched, I think we can then just take the max of `SQRT_2 * glyph.height()` and the old `scaled!(self, display_operator_min_height)` code.

Reply:
Thanks for explaination! I really have trouble understand different metrics here. :rofl: